packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -182,15 +197,6 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
         return;
       }
       focusInput();
-    },
-    onPress: (e) => {
-      if (e.pointerType === 'touch') {
-        focusInput();
-      } else if (e.pointerType !== 'virtual') {
-        if (triggerRef?.current) {
-          focusSafely(triggerRef.current);
-        }
-      }
```

Comment:
This section of code seemed extraneous, not sure what it was handling. Will dig more if we decide to go this route, but removing for now since it interferes with entering Edit mode via Enter

Reply:
If i recall correctly, this keeps the resizer from disappearing whenever the user lifts their finger in touch

also returns focus to the trigger for keyboard if you press Enter or lift your mouse

Reply:
Makes sense, I just realized I misread the `focusInput` part of that code block. Think I can change the `else if` to also omit `pointerType === keyboard` since it is handled in the onKeydown passed to useKeyboard.